### PR TITLE
Application Integration dashboard, containing charts for Integration Execution Count metric.

### PR DESCRIPTION
**Pull request by ShubhamThakkar**

### CODE REVIEW

File: dashboards/google-application-integration/integration-executions.json [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+                    "alignmentPeriod": "60s",
+                    "crossSeriesReducer": "REDUCE_SUM",
+                    "groupByFields": [],
+                    "perSeriesAligner": "ALIGN_SUM"
+                  },
+                  "filter": "metric.type=\"integrations.googleapis.com/integration/execution_count\" resource.type=\"integrations.googleapis.com/IntegrationVersion\" metric.label.\"status\"=\"SUCCEEDED\""
+                }
+              }
+            },
+            "title": "Total Integrations Succeeded"
```
Comment (EvanSimpson) [inline dashboards/google-application-integration/integration-executions.json:234]: Does this title (as well as the next 3) need to mention _executions_, or is this a count of integrations?

**Review by EvanSimpson — CHANGES_REQUESTED** [review]
Can you also add a README file for the new directory and the content below? We are working to remove this requirement, but it is still necessary in order to appear in the cloud console.

```
### Dashboards for Application Integration

|Integration Executions|
|:------------------|
|Filename: [integration-executions.json](integration-executions.json)|
|This dashboard has charts displaying count of Integrations executions with different status. All charts in this dashboard are based on the metrics "integrations.googleapis.com/integration/execution_count".|
```

File: dashboards/google-application-integration/integration-executions.json [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+      },
+      {
+        "filterType": "RESOURCE_LABEL",
+        "labelKey": "version",
+        "stringValue": "",
+        "templateVariable": ""
+      }
+    ],
+    "displayName": "Integration Executions",
+    "etag": "4fe2c9a94b9c86a9f84b407d9c062fe2",
```
[the remark this answers is not in the document]
Comment (ShubhamThakkar) [reply dashboards/google-application-integration/integration-executions.json]: Removed.

File: dashboards/google-application-integration/integration-executions.json (hunk 1 above)
Comment (ShubhamThakkar) [reply dashboards/google-application-integration/integration-executions.json:233]: Yes, this is needed, since it is count of Integrations.

**Comment by ShubhamThakkar** [comment]
> Can you also add a README file for the new directory and the content below? We are working to remove this requirement, but it is still necessary in order to appear in the cloud console.
> 
> ```
> ### Dashboards for Application Integration
> 
> |Integration Executions|
> |:------------------|
> |Filename: [integration-executions.json](integration-executions.json)|
> |This dashboard has charts displaying count of Integrations executions with different status. All charts in this dashboard are based on the metrics "integrations.googleapis.com/integration/execution_count".|
> ```

Added, thanks for pointing it out and thanks for reviewing the PR.

**Pull request merged**